metadata.yaml

```diff
@@ -3,12 +3,13 @@ display-name: Content-cache
 description: |
   A service providing local caches of HTTP servers and useful for building content delivery networks (CDN).
 summary: |
-  A service providing local caches of HTTP servers and useful for building content delivery networks (CDN).
+  A content cache for k8s charms.
```

Comment:
I don't think the caching is k8s specific. Maybe this would be more adequate? `HTTP cache service useful for building CDNs.`